docs/source/index.rst

```diff
@@ -0,0 +1,79 @@
+.. raw:: html
```

Comment:
Here I use `raw html`, but that should be further changed, I used that as I quickly copy/pasted things from `index.html`

Now that's it's a RST, I can have crosslink (almost) for free, and can remove most of the templates hacks that are hard to maintain.